sys-process/rtirq/rtirq-20150216.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
+HOMEPAGE="http://www.rncbc.org/jack/"
+
+SRC_URI="http://www.rncbc.org/jack/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=sys-apps/util-linux-2.13"
+
+src_prepare() {
+	# Correct config file path.
+	sed -i -e "s:^\(RTIRQ_CONFIG\=\)\(.*\):\1/etc/conf.d/rtirq:" ${PN}.sh || die
+	sed -i -e "s:/etc/sysconfig/rtirq:/etc/conf.d/rtirq:" ${PN}.conf || die
+
+	default
+}
+
+src_install(){
+	dosbin ${PN}.sh
+	doinitd "${FILESDIR}"/${PN}
+	newconfd ${PN}.conf ${PN}
+}
+
+pkg_postinst(){
+	if [ "$(rc-config list default | grep rtirq)" = "" ] ; then
```

Comment:
Hmm this is sh syntax AFAIK. Let's be a bit more pedantic and switch to the bash syntax with double [[ ]]'s.